### PR TITLE
US epic test

### PR DESCRIPTION
**Pull request by joelochlann**
please click on images to see at higher res!

| control | freedom_of_the_press | factual_truthful_reporting | leading_with_climate |
| - | - | - | - |
| picture 379 | freedom_of_the_press | factual_truthful_reporting | leading_with_climate_yellow_button |

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-us-leading-with-climate.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// @flow
+import { appendToLastElement } from 'lib/array-utils';
+
+const gridImage = `
+    <div class="u-responsive-ratio" style="padding-bottom: 60.01%;">
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-us-leading-with-climate.js]: I took this directly from the HTML for the image at the top of this article, which is about the same size: https://www.theguardian.com/us-news/2018/nov/09/california-wildfire-paradise-chico-camp-fire

There are a bunch of abstractions around the grid stuff which provide all these different `srcset` images at different resolutions for the right screen size, but these are all serverside. So I thought I'd just copy-paste the whole lot to use them clientside.

**Comment by ionamckendrick** [comment]
Looking great. 
One typo (my fault) in the freedom of the press variant. Last sentence should drop a spare "the", and read "Support the free press by..."

**Comment by joelochlann** [comment]
FREE THE PRESS!!!

**Comment by ionamckendrick** [comment]
Also, the yellow highlight eats into some of the letters (feels like more so than usual). Could we play with this? Even if it means an increase in line height for that last part.

**Comment by joelochlann** [comment]
There was some debate about whether we should actually change the hook in the control without testing it. I say we revisit the issue in the morning and see what Amanda and @jranks123 think

**Comment by ionamckendrick** [comment]
We also need to have a black button (#121212) across all please!

**Comment by ionamckendrick** [comment]
Oh the changing of the hook, Amanda and I have decided we will KEEP "We have some news..." here as the control. Sorry, you will have grabbed the copy before that decision. I've altered this in our doc, so if you wouldn't mind updating that would be great @joelochlann

**Comment by joelochlann** [comment]
> Also, the yellow highlight eats into some of the letters (feels like more so than usual). Could we play with this? Even if it means an increase in line height for that last part.

Yeah, I remember having this discussion a few times in the past. Can't remember why we upped it since it definitely looks better at `2px` all round.

Have taken it to `2px`

it now looks like this

picture 366
picture 365

**Comment by joelochlann** [comment]
> We also need to have a black button (#121212) across all please!

what? when did that change???

**Comment by ionamckendrick** [comment]
New highlight looks great, thanks!

**Review by jranks123 — APPROVED** [review]
All good! A few comments, nothing blocking

**Pull request merged**